contrib/devtools/interactive-test-runner.py

```diff
@@ -105,8 +107,12 @@ def run_command(*args, **kwargs):
     def find_git_root(self):
         return self.run_command('git', 'rev-parse', '--show-toplevel')
 
-    def find_functional_tests(self):
-        stdout = self.run_command('git', 'ls-files', '*.py', cwd=self.functional_test_root)
+    def find_functional_tests(self, only_modified=False):
+        args = 'git ls-files'.split()
```

Comment:
nit: Why not using the literal `['git', 'ls-files']` ?

Reply:
Thought it was more readable... I take it as a nit and follow-up on this later.